test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
+    PARAMS+="rd.hostonly=0"
+else
+    PARAMS+="ro"
```

Comment:
We'd be probably better off dropping `ro` for all, but this can be done as a separate PR.